app/controllers/movie_actors_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MovieActorsController < ApplicationController
+  def create
+    movie = Movie.find(params[:movie_id])
+    movie_actor = MovieActor.create!(mov_act_params)
+
+    redirect_to "/movies/#{movie.id}"
+  end
+
+  private
+  def mov_act_params
```

Comment:
This is probably more a personal opinion of mine that any firm thing that needs to change, but I don't love abbreviations in method and variable names. This is a small app so it's easy to figure out what this means. Once apps start getting bigger and more complex, it's a lot more mental load keeping track of things that are unnecessarily ambiguous. I'd probably name this `movie_actor_params` and just spell the whole thing out.

Reply:
Thanks for this note!